### DISCUSSION TITLE
What's the suggested fingerprint for HTTP handlers?

### DISCUSSION THREAD

**Discussion by agis** [getsentry/sentry-go#359] [Q&A]
Hi all!

Using the default grouping, I've been experiencing different underlying events/error causes being grouped under the same Sentry issue. So I'm wondering, is there any suggested or commonly used fingerprint for use in HTTP handlers?

Thanks in advance.

**Comment by rhcarvalho** [comment]
Hi @agis, I'd look into the event data of two errors that were grouped together but should be separate and that should show what we can use to generate fingerprints.

Would you have a link to such a pair in Sentry.io or at least a code example of how you capture errors that end up grouped together?

**Reply by agis** [reply]
Unfortunately I don't have anything to share currently, because it's been a while since we moved away from the default fingerprint. But I can try switching back to the default one and see if things are getting better.

So AFAICT the suggested fingerprint is to use `{{ default }}`, which is effectively done by not setting a custom fingerprint at all in the driver?

**Reply by rhcarvalho** [reply]
Right, we also have some new grouping strategies.

Have you checked your project settings at `https://sentry.io/settings/<YOUR_ORG>/projects/<YOUR_PROJECT>/issue-grouping/`?

Also https://docs.sentry.io/platforms/go/data-management/event-grouping/fingerprint-rules/
